### PR TITLE
Created & updated PromptDetails, ImageCarousel

### PR DESCRIPTION
**Pull request by leovaldez08**
Finished components for listing the details of the prompt. Also created carousel to display the images generated by the prompt.

### CODE REVIEW

File: FrontEnd/src/Components/ImageCarousel.jsx [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+//             }}
+//             onClick={() => setSlide(slide)}
+//           ></Box>
+//         ))}
+//       </HStack>
+//     </Box>
+//   );
+// };
+
+// export default ImageCarousel;
```
Comment (jithin-kalvian) [inline FrontEnd/src/Components/ImageCarousel.jsx]: remove the commented code

File: FrontEnd/src/Components/ImageCarousel.jsx [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+      img: "https://images.pexels.com/photos/2714581/pexels-photo-2714581.jpeg?auto=compress&cs=tinysrgb&dpr=2&h=650&w=940",
+    },
+    {
+      img: "https://images.pexels.com/photos/2878019/pexels-photo-2878019.jpeg?auto=compress&cs=tinysrgb&dpr=3&h=750&w=1260",
+    },
+    {
+      img: "https://images.pexels.com/photos/1142950/pexels-photo-1142950.jpeg?auto=compress&cs=tinysrgb&dpr=2&h=650&w=940",
+    },
+    {
+      img: "https://images.pexels.com/photos/3124111/pexels-photo-3124111.jpeg?auto=compress&cs=tinysrgb&dpr=2&h=650&w=940",
```
Comment (jithin-kalvian) [inline FrontEnd/src/Components/ImageCarousel.jsx]: create a js file and import the mock data

**Review by jithin-kalvian — CHANGES_REQUESTED** [review]
please make the requested changes

File: FrontEnd/src/Components/ImageCarousel.jsx (hunk 1 above)
Comment (leovaldez08) [reply FrontEnd/src/Components/ImageCarousel.jsx]: Done 👍

File: FrontEnd/src/Components/ImageCarousel.jsx (hunk 2 above)
Comment (leovaldez08) [reply FrontEnd/src/Components/ImageCarousel.jsx]: Done 👍

**Review by yash0212 — APPROVED** [review]
LGTM

**Pull request merged**